pkg/sql/logictest/testdata/logic_test/builtin_function

```diff
@@ -1928,6 +1928,16 @@ SELECT length(gen_random_uuid()::BYTES), gen_random_uuid() = gen_random_uuid()
 ----
 16 false
 
+query IB
+SELECT length(gen_random_ulid()::BYTES), gen_random_ulid() = gen_random_ulid()
+----
+16 false
+
+query TTTTTT
+SELECT uuid_ulid_to_string('0178951c-b665-30a7-19a2-a18999834858')
+----
+01F2AHSDK562KHK8N1H6CR6J2R
+
```

Comment:
Also can we add a test for monotonicity.

Reply:
Not sure how to do this in a clean way only using this input/output text file, without writing any specific test (maybe I should do this)

I've been looking here:

https://github.com/oklog/ulid/blob/0d4fda9d6345755e157a256fd33d48556c5f4a7a/ulid_test.go#L520-L560

Any ideas?

Reply:
There's this neat `let` feature that lets you store the output for a command.
https://github.com/cockroachdb/cockroach/blob/master/pkg/sql/logictest/testdata/logic_test/sequences_regclass#L100

We can create two ULIDs and use a string comparison to ensure the ordering is correct.

Reply:
This is great info, I updated the code making use of let :smile: